core/arch/aarch64/codec.c

```diff
@@ -2341,7 +2363,8 @@ static inline uint
 encode_opnds_b(byte *pc, instr_t *instr, uint enc)
 {
     uint off;
-    if (instr_num_dsts(instr) == 0 && instr_num_srcs(instr) == 1 &&
+    if (((instr_get_opcode(instr) == OP_bl && instr_num_dsts(instr) == 1) ||
+        instr_num_dsts(instr) == 0) && instr_num_srcs(instr) == 1 &&
```

Comment:
I've put instr_num_srcs()  on a new line. I've also changed the alignment slightly to match the outermost applying bracket. Not sure if that's correct though :(